mteb/tasks/PairClassification/multilingual/RTE3.py

```diff
@@ -0,0 +1,88 @@
+from __future__ import annotations
+
+import datasets
+
+from mteb.abstasks import MultilingualTask
+from mteb.abstasks.AbsTaskPairClassification import AbsTaskPairClassification
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+_LANGS = {
+    "de": ["deu-Latn"],
+    "en": ["eng-Latn"],
+    "fr": ["fra-Latn"],
+    "it": ["ita-Latn"],
+}
+
+
+class RTE3(MultilingualTask, AbsTaskPairClassification):
+    metadata = TaskMetadata(
+        name="RTE3",
+        dataset={
+            "path": "maximoss/rte3-multi",
+            "revision": "d94f96ca5a6798e20f5a77e566f7a288dc6138d7",
+        },
+        description="Recognising Textual Entailment Challenge (RTE-3) aim to provide the NLP community with a benchmark to test progress in recognizing textual entailment",
+        reference="https://aclanthology.org/W07-1401/",
+        category="s2s",
+        type="PairClassification",
+        eval_splits=["test", "validation"],
+        eval_langs=_LANGS,
+        main_score="ap",
+        date=("2023-03-25", "2024-04-15"),
+        form=["written"],
+        domains=None,
+        task_subtypes=["Claim verification"],
+        license="cc-by-4.0",
+        socioeconomic_status=None,
+        annotations_creators="expert-annotated",
+        dialect=[],
+        text_creation="created",
+        bibtex_citation="""@inproceedings{giampiccolo-etal-2007-third,
+            title = "The Third {PASCAL} Recognizing Textual Entailment Challenge",
+            author = "Giampiccolo, Danilo  and
+            Magnini, Bernardo  and
+            Dagan, Ido  and
+            Dolan, Bill",
+            booktitle = "Proceedings of the {ACL}-{PASCAL} Workshop on Textual Entailment and Paraphrasing",
+            month = jun,
+            year = "2007",
+            address = "Prague",
+            publisher = "Association for Computational Linguistics",
+            url = "https://aclanthology.org/W07-1401",
+            pages = "1--9",
+        }
+        """,
+        n_samples={"validation": 3200, "test": 3200},
```

Comment:
Do we need both validation and test? Should we reduce test to 2048?

Reply:
Only keeping test sounds fine.
It's 3200 total and 800 per language, should I put 800?

Reply:
Just removed the validation and updated the number to 800 for test.

Reply:
For multilingual tasks, we computed the number of samples by suming up all subsets of languages. Should be fine to have 3200.

Reply:
switched back to 3200. Since the sampling is done at the language level (checked this for XNLI), and each language has 800 samples, I did not apply it.

Reply:
It sounds good with just the test set and the 3200 samples

Reply:
ended up updating it to 1923 which is the actual total on the 4 languages when the neutral examples are filtered out. Updated the average char length accordingly